packages/blocks/src/api/validation/index.js

```diff
@@ -325,6 +332,21 @@ export function isEquivalentTextTokens(
 	return false;
 }
 
+/**
+ * Given a CSS length value, returns a normalized CSS length value for strict equality
+ * comparison.
+ *
+ * @param {string} value CSS length value.
+ *
+ * @return {string} Normalized CSS length value.
+ */
+export function getNormalizedLength( value ) {
+	if ( CSS_UNITS.some( ( unit ) => value === '0' + unit ) ) {
+		return '0';
+	}
+	return value;
```

Comment:
Why would we limit this to specific units? Would it not be the same if we did something like this?
```js
return 0 === parseFloat( value ) ? 0 : value;
```
`0pt`, `0ch` and all other units are also valid, wouldn't using `parseFloat` be safer and future-proof?

Reply:
I wasn't aware that [`parseFloat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseFloat) cuts off trailing strings :grimacing: 

Anyway, thanks for your suggestion, makes sense to me. I'll change it accordingly!